activefires_pp/config.py

```diff
@@ -33,3 +32,11 @@ def read_config(config_filepath):
         config = yaml.load(fp_, Loader=UnsafeLoader)
 
     return config
+
+
+def get_xauthentication_token(xauth_filepath):
+    """Get the X-Authentication-token needed for posting to the API."""
+    with open(xauth_filepath, 'r') as fp_:
+        tokens = yaml.load(fp_, Loader=UnsafeLoader)
```

Comment:
Why UnsafeLoader? there is nothing python to be read there, right?

Reply:
Ah, I see, you are right.

Reply:
Fixed